dev/MRTCore/azure-pipelines.yml

```diff
@@ -234,12 +234,12 @@ jobs:
       artifactName: 'binaries_raw'
       downloadPath: '$(Build.ArtifactStagingDirectory)\raw'
   
-  - task: PkgESCodeSign@10
-    env:
-      SYSTEM_ACCESSTOKEN: $(System.AccessToken)
-    inputs:
-      signConfigXml: 'SignConfig.xml'
-      inPathRoot: '$(BUILD.ArtifactStagingDirectory)\raw\binaries_raw\lib'
+  #- task: PkgESCodeSign@10
```

Comment:
do you know why this isn't working in our pipeline?  we haven't turned on the signing code that we cloned from the winui2 repo yet.  Steve Kirbach is also looking into this right now.

Reply:
This won't work on the *public* pipelines because the pool being requested and tasks that this is using are only available on the private ADO organization. You can only do these jobs/tasks on the private org (these are "release pipeline" things, we don't sign our CI binaries).